docs/source/etl/lift-over.rst

```diff
@@ -66,7 +66,7 @@ Pull a target assembly reference file down to every node in the Spark cluster in
 performing variant liftover.
 
 The ``lift_over_variants`` transformer operates on a DataFrame containing genetic variants and supports the following
-:ref:`options <transformer-options>`.
+options:
```

Comment:
I think I resolved this in master - https://github.com/projectglow/glow/blob/master/docs/source/tertiary/pipe-transformer.rst

Reply:
Not sure why this "options" should refer to pipe transformer options.

Reply:
Perhaps it'd make sense to have a separate section describing transformers, as we expose the same Scala/Python APIs and share behavior such as case-sensitivity.

Reply:
Sure will do in a separate PR